external-crates/move/crates/move-compiler/src/linters/freezing_capability.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
This should be moved to sui-linters as it is Sui specific

Reply:
refactored